src/server.js

```diff
@@ -88,14 +154,113 @@ export class Server {
         .toXDR()
         .toString('base64')
     );
+
     return HorizonAxiosClient.post(
       URI(this.serverURL)
         .segment('transactions')
         .toString(),
       `tx=${tx}`,
       { timeout: SUBMIT_TRANSACTION_TIMEOUT }
     )
-      .then((response) => response.data)
+      .then((response) => {
+        if (!response.data.result_xdr) {
+          return response.data;
+        }
+
+        const responseXDR = xdr.TransactionResult.fromXDR(
+          response.data.result_xdr,
+          'base64'
+        );
+
+        const results = responseXDR.result().value();
+
+        let offerResults;
+        let hasManageOffer;
+
+        if (results.length) {
+          offerResults = results
+            .map((result, i) => {
+              if (typeof result.value().value().success !== 'function') {
```

Comment:
Would be so easy with https://github.com/stellar/js-xdr/issues/25 😉

Reply:
I agree but there isn't time left in the sprint to do that. Should be easy for whomever does close that issue to port this over.